go.mod

```diff
@@ -140,6 +139,7 @@ require (
 )
 
 replace (
+	github.com/banzaicloud/go-cruise-control => github.com/alex-necula/go-cruise-control v0.6.0-rc5 // TODO this will be replaced with adobe fork once it's created
```

Comment:
Update with new release once https://github.com/adobe/go-cruise-control/pull/1 is merged